pkgs/development/python-modules/kubernetes/default.nix

```diff
@@ -1,34 +1,32 @@
-{ lib, buildPythonPackage, fetchPypi, pythonAtLeast,
-  ipaddress, websocket_client, urllib3, pyyaml, requests_oauthlib, python-dateutil, google-auth, adal,
-  isort, pytest, coverage, mock, sphinx, autopep8, pep8, codecov, recommonmark, nose }:
+{ lib, buildPythonPackage, fetchPypi, pythonAtLeast
+, ipaddress, websocket_client, urllib3, pyyaml, requests_oauthlib, python-dateutil, google-auth, adal
+, isort, pytestCheckHook, coverage, mock, sphinx, autopep8, pep8, codecov, recommonmark, nose }:
 
 buildPythonPackage rec {
   pname = "kubernetes";
   version = "12.0.1";
 
-  prePatch = ''
-    sed -e 's/sphinx>=1.2.1,!=1.3b1,<1.4 # BSD/sphinx/' -i test-requirements.txt
-
-    # This is used to randomize tests, which is not reproducible. Drop it.
-    sed -e '/randomize/d' -i test-requirements.txt
-  ''
-  # This is a python2 and python3.2 only requiremet since it is a backport of a python-3.3 api.
-  + (if (pythonAtLeast "3.3")  then ''
-    sed -e '/ipaddress/d' -i requirements.txt
-  '' else "");
-
-  doCheck = pythonAtLeast "3";
-  checkPhase = ''
-    py.test --ignore=kubernetes/dynamic/test_client.py
-  '';
-
   src = fetchPypi {
     inherit pname version;
     sha256 = "ec52ea01d52e2ec3da255992f7e859f3a76f2bdb51cf65ba8cd71dfc309d8daa";
   };
 
-  checkInputs = [ isort coverage pytest mock sphinx autopep8 pep8 codecov recommonmark nose ];
+  # prePatch = ''
+  #   sed -e 's/sphinx>=1.2.1,!=1.3b1,<1.4 # BSD/sphinx/' -i test-requirements.txt
+
+  #   # This is used to randomize tests, which is not reproducible. Drop it.
+  #   sed -e '/randomize/d' -i test-requirements.txt
+  # ''
+  # # This is a python2 and python3.2 only requiremet since it is a backport of a python-3.3 api.
+  # + lib.optionalString (pythonAtLeast "3.3")
+  # "sed -e '/ipaddress/d' -i requirements.txt";
```

Comment:
What's the story with these? Do you think they're now redundant?

Reply:
most likely if the package isn't used in python 2 anymore.